build_runner/lib/src/server/server.dart

```diff
@@ -43,10 +43,17 @@ class AssetHandler {
 
   AssetHandler(this._reader, this._rootPackage);
 
-  Future<Response> handle(Request request) async {
+  Future<Response> handle(Request request) {
     var pathSegments = request.url.pathSegments;
+    if (request.url.path.endsWith('/')) {
```

Comment:
Pub does not require this (and I don't  think most servers generally do).

Reply:
synced offline. Dartium doesn't work visiting `host:port/web` but does visiting `host:port/web/`. We should probably stick with this for now and revisit if we find a need.